hedera-node/hedera-app/src/main/java/com/hedera/node/app/SessionContext.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app;
+
+// Put the per-thread stuff here like parsers
```

Comment:
Is there a common convention for TODOs ? I started using `FUTURE` instead of `TODO`.

Reply:
It'd be nice to have something Intellij can key off of. `FUTURE` sounds good to me

Reply:
Sounds good